src/main/java/frc/robot/commands/Presets/HighPresetSequence.java

```diff
@@ -32,5 +33,6 @@ public HighPresetSequence(TelescopeSubsystem telescope, RotatorSubsystem rotator
             crocodile.setWristToPositionCommand(WristPosition.CONE_SCORE).withTimeout(2)
             
         );
+        deadlineWith(new InstantCommand(() -> crocodile.setIntakeSpeed(MathUtil.clamp((0.25 + intake.getAsDouble()), -1, 1) * multiplier)));
```

Comment:
This should be a `RunCommand` not an `InstantCommand`!!!

Reply:
Right, will fix